iphone/Classes/TiAppiOSUserActivityProxy.m

```diff
@@ -197,6 +197,7 @@ -(NSNumber*)supported
 - (void)userActivityWillSave:(NSUserActivity *)userActivity
 {
     if([self _hasListeners:@"useractivitywillsave"]){
+        DebugLog(@"[WARN] Titanium.App.IOS.UserActivity.useractivitywillsave event is deprecated. Update user activity by setting Titanium.App.IOS.UserActivity.needsSave property to true instead.");
```

Comment:
Just setting needsSave is not gonna do it. I think it's better to put it this way:

> Update user activity and then set the Titanium.App.iOS.UserActivity.needsSave property to true every time you have a change.